<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateSetupDialog</class>
 <widget class="QDialog" name="CalibrateSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibrateSetupDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>315</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>315</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbHgb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>165</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>HGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWbc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>75</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>WBC</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRbc">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RBC</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leWbc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>75</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leRbc">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leHgb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>165</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="lePlt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>255</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbMcv">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MCV</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>255</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PLT</string>
   </property>
  </widget>
  <widget class="DigitLineEdit" name="leMcv">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLotNo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot No</string>
   </property>
  </widget>
  <widget class="LineEdit" name="leLotNo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DigitLineEdit</class>
   <extends>LineEdit</extends>
   <header>digitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
